packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -161,7 +161,10 @@ class _DropdownMenuItemButtonState<T> extends State<_DropdownMenuItemButton<T>>
   }
 
   static final Map<LogicalKeySet, Intent> _webShortcuts =<LogicalKeySet, Intent>{
-    LogicalKeySet(LogicalKeyboardKey.enter): const ActivateIntent(),
```

Comment:
@gspencergoog I found out that this isn't necessary since we already have the `ButtonActivateIntent` in here. Let me know if this isn't correct and I'll be happy to put back `LogicalKeyboardKey.enter`.

Reply:
Yes, I think you're right.